css/contact-us.css

```diff
@@ -340,11 +366,11 @@ textarea.form-control__input {
   background-color: blue;
```

Comment:
.form-checkbox__box::hover - single colon sign for HOVER pseudo-class please.
line 357 and line 361

Reply:
fixed

Reply:
fixed